api/clients/v2/payloaddispersal/payload_disperser.go

```diff
@@ -283,6 +367,12 @@ func (pd *PayloadDisperser) pollBlobStatusUntilSigned(
 	initialStatus dispgrpc.BlobStatus,
 	probe *common.SequenceProbe,
 ) (*dispgrpc.BlobStatusReply, error) {
+	ctx, span := tracer.Start(ctx, "PayloadDisperser.pollBlobStatusUntilSigned",
```

Comment:
You're creating new spans for each of these utility methods. Does that actually make sense? Why not just pass the span through as a parameter?

Reply:
This is the opentelemetry suggested pattern https://opentelemetry.io/docs/languages/go/instrumentation/#create-nested-spans

Reply:
As for whether or not we should be creating a span for each utility method it really depends if it's worth tracking the timing / attributes associated with it and it's something we can adjust depending on what we want to observe. Here is an example span:
<img width="3206" height="1132" alt="image" src="https://github.com/user-attachments/assets/9152afb8-9c81-4bca-8a8e-ef61e47c772d" />